core/unit_test/tools/TestProfilingMacros.cpp

```diff
@@ -0,0 +1,7 @@
+#include <Kokkos_Core.hpp>
+#include <Kokkos_Macros.hpp>
+#include <iostream>
+int main() {
+  auto name = KOKKOS_FILELINE_NAME(dogs);
+  std::cout << name << std::endl;
```

Comment:
Why would you not assert directly within your unit test?

Reply:
I'm not really sure what the preferred method is, I'm certainly game to change it to that

Reply:
Damien and I talked this through in another channel, it's reimplemented a different way

Reply:
You're the only one that introduced these regular expressions into testing. :slightly_smiling_face: